packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -391,12 +394,29 @@ export const getCacheUpdateMatrix = (customerId: string | null) => {
     }
 }
 
+export const SHOPPER_BASKETS_NOT_IMPLEMENTED = [
+    'addGiftCertificateItemToBasket',
+    'addPriceBooksToBasket',
+    'addTaxesForBasket',
+    'addTaxesForBasketItem',
+    'createShipmentForBasket',
+    'removeGiftCertificateItemFromBasket',
+    'removeItemFromBasket',
+    'removeShipmentFromBasket',
+    'transferBasket',
+    'updateGiftCertificateItemInBasket',
+    'updateShipmentForBasket'
+]
+
 /**
  * A hook for performing mutations with the Shopper Baskets API.
  */
 export function useShopperBasketsMutation<Action extends ShopperBasketMutationType>(
     action: Action
 ) {
+    if (SHOPPER_BASKETS_NOT_IMPLEMENTED.includes(action)) {
+        NotImplementedError()
+    }
```

Comment:
Do we have a plan for how this is handled from a test coverage % perspective? We should make sure to ignore lapses in coverage for things we're actively not covering. Can be a separate ticket / PR